odoo/http.py

```diff
@@ -2783,10 +2783,12 @@ def fake_start_response(status, headers):
                     pass
                 elif isinstance(exc, SessionExpiredException):
                     _logger.info(exc)
-                elif isinstance(exc, (UserError, AccessError)):
+                elif isinstance(exc, AccessError) and 'access' in config['dev_mode']:
+                    _logger.warning(exc, exc_info=True)
```

Comment:
Shouldn't it be :
```suggestion
                elif isinstance(exc, AccessError):
                    _logger.warning(exc, exc_info='access' in config['dev_mode'])
```
Or is it intended to not have the log anymore when this dev setting is disabled ?

Reply:
`isinstance(exc, UserError)` catch the AccessError, but your code is simpler, doing the change, thanks